src/sentry/eventstream/kafka/backend.py

```diff
@@ -282,10 +340,12 @@ def run_post_process_forwarder(
         commit_batch_timeout_ms: int,
         concurrency: int,
         initial_offset_reset: Union[Literal["latest"], Literal["earliest"]],
+        strict_offset_reset: bool,
```

Comment:
Nit: The number of arguments to this api has really grown over time. Maybe it would be worth refactoring some of the code to reduce the number of arguments which need to be passed around

Reply:
Good point, I am going to tackle this later after the new consumer is tested and we are confident about it. There will be a lot of stuff we can remove them.